src/components/dashboard/dashboard.spec.ts

```diff
@@ -121,4 +123,128 @@ describe('Dashboard', () => {
 
 function getPixels(cssValue: string): number {
     return parseInt(cssValue.replace('px', ''));
-}
\ No newline at end of file
+}
+
+@Component({
+    selector: 'app-ux-dashboard-layout',
+    template: `<ux-dashboard
+                    [options]="options"
+                    [(layout)]="layout"
+                    (layoutChange)="onLayoutChange()">
+
+                    <ux-dashboard-widget
+                        id="run-widget"
+                        name="Runs">
+                    </ux-dashboard-widget>
+
+                    <ux-dashboard-widget
+                        id="purpose-widget"
+                        name="Purpose">
+                    </ux-dashboard-widget>
+
+                    <ux-dashboard-widget
+                        id="host-widget"
+                        name="Host">
+                    </ux-dashboard-widget>
+
+            </ux-dashboard>
+`
+})
+export class DashboardComponentLayout {
```

Comment:
I suggest DashboardWithInitialLayoutTestComponent here. The more precise you can be with identifiers the better.